pkg/pool/pool.go

```diff
@@ -225,3 +229,90 @@ func (p *pool) SearchObject(ctx context.Context, params *client.SearchObjectPara
 	}
 	return conn.SearchObject(ctx, params, options...)
 }
+
+func (p *pool) PutContainer(ctx context.Context, cnr *container.Container, option ...client.CallOption) (*cid.ID, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.PutContainer(ctx, cnr, options...)
+}
+
+func (p *pool) GetContainer(ctx context.Context, cid *cid.ID, option ...client.CallOption) (*container.Container, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.GetContainer(ctx, cid, options...)
+}
+
+func (p *pool) ListContainers(ctx context.Context, ownerID *owner.ID, option ...client.CallOption) ([]*cid.ID, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.ListContainers(ctx, ownerID, options...)
+}
+
+func (p *pool) DeleteContainer(ctx context.Context, cid *cid.ID, option ...client.CallOption) error {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return err
+	}
+	return conn.DeleteContainer(ctx, cid, options...)
+}
+
+func (p *pool) GetEACL(ctx context.Context, cid *cid.ID, option ...client.CallOption) (*client.EACLWithSignature, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.GetEACL(ctx, cid, options...)
+}
+
+func (p *pool) SetEACL(ctx context.Context, table *eacl.Table, option ...client.CallOption) error {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return err
+	}
+	return conn.SetEACL(ctx, table, options...)
+}
+
+func (p *pool) AnnounceContainerUsedSpace(ctx context.Context, announce []container.UsedSpaceAnnouncement, option ...client.CallOption) error {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return err
+	}
+	return conn.AnnounceContainerUsedSpace(ctx, announce, options...)
+}
+
+func (p *pool) CheckIsContainerCreatedPoll(ctx context.Context, cid *cid.ID, option ...client.CallOption) (bool, error) {
+	const (
+		containerCreationTimeout = 120 * time.Second
+		containerPollInterval    = 5 * time.Second
```

Comment:
These better be visible to the outside world (for documentation purposes) and overridable with parameters.

Reply:
Moved the consts into a struct